Added more SMTP setup steps to Enterprise installation guides (#247)
astronomer/docs: enterprise/v0.23/03_install/01_aws/01_install-aws-standard.md

To install Astronomer on EKS, you'll need access to the following tools and permissions:

- The [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)

> - Kubernetes 1.16 or later

Reviewer: ```suggestion
- Kubernetes 1.16, 1.17, or 1.18
```

For the base domain `astro.mydomain.com`, for example, here are some corresponding URLs that your users would be able to reach:

- Astronomer UI: `app.astro.mydomain.com`

> - Airflow Deployments: `deployments.astro.mydomain.com/udeployment-release-name/airflow`

Reviewer: ```suggestion
- Airflow Deployments: `deployments.astro.mydomain.com/deployment-release-name/airflow`
```

eep, typo - might have been me

> - Airflow Deployments: `deployments.astro.mydomain.com/uniquely-generated-airflow-name/airflow`

Reviewer: ```suggestion
- Airflow Deployments: `deployments.astro.mydomain.com/deployment-release-name/airflow`
```

As you follow the guide linked above, keep in mind:

> - Astronomer currently supports Kubernetes versions 1.14, 1.15, 1.16 and 1.17 on EKS.

Reviewer: ```suggestion
- Astronomer currently supports Kubernetes versions 1.16, 1.17, and 1.18 on EKS. For more information, refer to [Version Compatibility Reference](https://www.astronomer.io/docs/enterprise/v0.23/resources/version-compatibility-reference).
```

1. Add the root certificate provided by your security team to an [Opaque Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/#secret-types) in the Astronomer namespace by running the following command:

```sh
$ kubectl create secret generic private-root-ca --from-file=cert.pem=./<your-certificate-filepath>
```

> **Note:** The root certificate which you specify here should be the certificate of the authority that signed the Astronomer certificate, rather than the Astronomer certificate itself. This is the same certificate you need to install with all clients to get them to trust your services.

2. Note the value of `private-root-ca` for when you configure your Helm chart in Step 8. You'll need to additionally specify the `privateCaCerts` key-value pair with this value for that step.

## Step 6: Retrieve Your SMTP URI

An SMTP service is required so that users can send and accept email invites to and from Astronomer. To integrate your SMTP service with Astronomer, make note of your SMTP service's URI and add it to your Helm chart in Step 8. In general, an SMTP URI will take the following form:

```
smtps://USERNAME:PASSWORD@HOST/?pool=true
```

> The following table contains examples of what the URI will look like for some of the most popular SMTP services. If your SMTP provider is not listed, refer to the provider's documentation for information on creating an SMTP URI:

Reviewer: I think in `/next` we have this line at the bottom of the graph - makes more sense there?
